### ISSUE TITLE
ionic serve error "Cannot get /"

### ISSUE THREAD

**Issue by LuizTokuhara** [ionic-team/ionic#20432]
# Bug Report

**Ionic version:**

Ionic: 4.11.10

**Current behavior:**

Fresh ionic template is getting 404 error

**Expected behavior:**

**Steps to reproduce:**

First create a new ionic project (ionic start testApp blank).
Select Angular and select to not install Capacitor.
Later serve on browser (ionic serve).
Error happens.

[ng] ERROR in src/app/app.component.ts:4:30 - error TS2307: Cannot find module '@ionic-native/splash-screen/ngx'.
[ng] 4 import { SplashScreen } from '@ionic-native/splash-screen/ngx';
[ng]                                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
[ng] src/app/app.module.ts:6:30 - error TS2307: Cannot find module '@ionic-native/splash-screen/ngx'.
[ng] 6 import { SplashScreen } from '@ionic-native/splash-screen/ngx';

**Related code:**

```
insert short code snippets here
```

**Other information:**

**Ionic info:**

```
   Ionic CLI                     : 6.0.1 (/usr/local/lib/node_modules/@ionic/cli)
   Ionic Framework               : @ionic/angular 4.11.10
   @angular-devkit/build-angular : 0.801.3
   @angular-devkit/schematics    : 8.1.3
   @angular/cli                  : 8.1.3
   @ionic/angular-toolkit        : 2.1.2

Utility:

   cordova-res (update available: 0.9.0) : 0.3.0
   native-run (update available: 0.3.0)  : 0.2.6

System:

   NodeJS : v10.16.0 (/usr/local/bin/node)
   npm    : 6.9.0
   OS     : macOS Mojave
```

**Comment by NikolaPeevski** [comment]
@LuizTokuhara Can confirm it's a bug, just remove the last '/ngx' from app.module and app.component and it will build without any issues
@liamdebeasi I can't really submit a PR to fix this because the template is hosted at https://d2ql0qc7j8u4b2.cloudfront.net/ , someone of access can only address this :(

**Comment by liamdebeasi** [comment]
Thanks for the issue. This was a problem with Ionic Native. A new version has been published that should fix this issue (v5.21.4).
